java/java.lsp.server/vscode/src/extension.ts

```diff
@@ -356,6 +356,27 @@ export function activate(context: ExtensionContext): VSNetBeansAPI {
             await commands.executeCommand(selected.userData.command.command, ...(selected.userData.command.arguments || []));
         }
     }));
+    const mergeWithLaunchConfig = (dconfig : vscode.DebugConfiguration) => {
```

Comment:
Excellent ! This will perfectly work with the project view's actions (run project, debug project).